messages_appendix.cddl

```diff
@@ -549,7 +505,159 @@ receive-video-capability = {
   ? 8: bool ; supports-scaling
 }
 
-receive-data-capability = {
-  0: format ; format
+; type key 134
+streaming-session-start-request = {
+  request
+  1: uint; streaming-session-id
+  2: [* media-stream-offer] ;  stream-offers
+  3: microseconds ; stats-interval
+}
+
+; type key 135
+streaming-session-start-response = {
+  response
+  1: &result ; result
+  2: [* media-stream-request] ; stream-requests
+  3: microseconds ; stats-interval
+}
+
+; type key 136
+streaming-session-modify-request = {
+  request
+  1: uint ; streaming-session-id
+  2: [* media-stream-request] ; stream-requests
+}
+
+; type key 137
+streaming-session-modify-response = {
+  response
+  1: &result ; result
+}
+
+; type key 138
+streaming-session-terminate-request = {
+  request
+  1 : uint ; streaming-session-id
+}
+
+; type key 139
+streaming-session-terminate-response = {
+  response
+}
+
+; type key 140
+streaming-session-terminate-event = {
```

Comment:
I didn't see this message referenced in the spec.

Reply:
It's in this paragraph: "Finally, the sender may terminate the streaming session by sending
a streaming-session-terminate-request command.  When the receiver
receives the streaming-session-terminate-request, it should send back
a streaming-session-terminate-response.  The receiver can terminate at
any point and notify the sender by sending a
streaming-session-terminate-event message."

Reply:
Got it.  After this lands, we can go through and link all the message definitions which helps them stand out.